_pkgdown.yml

```diff
@@ -46,10 +46,9 @@ reference:
       - get_join_keys
       - get_metadata
       - tdata2env
-  - title: Functions moved to other packages
-    desc: These functions have been moved from teal and will be deprecated.
-    contents:
-      - log_app_usage
+  # - title: Functions moved to other packages
```

Comment:
lets just remove these?

Reply:
I decided to leave it in case it comes up again to keep the format of the `pkgdown` output.